packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -239,11 +269,16 @@ export function registerLocalNetwork(
 }
 
 export function isNetwork(chainId: ChainId) {
+  const customChains = getCustomChainsFromLocalStorage().filter(
+    chain => chain.chainID === chainId
+  )
```

Comment:
We have similar logic in multiple place, can we extract it to a function?

Reply:
Also, do we want to run the localStorage every time? Or can we parse it only once when starting the app?

Reply:
I extracted it to `getCustomChainFromLocalStorageById`. Af for running localStorage every time, we don't do it in quick successions so I don't think it's much of an issue?